build.gradle.kts

```diff
@@ -164,8 +170,14 @@ tasks.register(
   )
 }
 
+tasks.register<Exec>("install-commons") {
+  val buildCommons = providers.gradleProperty("buildCommons")
+  onlyIf("To build commons library run gradle build -PbuildCommons") { buildCommons.isPresent }
```

Comment:
Why is there a different profile for building with commons?

Reply:
If not enabled, all builds will checkout ecommerce commons and perform maven build before module ones, also local ones.

This way I want to replicate the maven approach with mvn validate/ mvn package